api/src/public/setup/setup.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>The CHT is starting up</title>
+  <script>
+    const getListElem = () => document.getElementById('actions');
+    const clearList = () => {
+      const actionsElem = getListElem();
+      while (actionsElem.firstChild) {
+        actionsElem.removeChild(actionsElem.firstChild);
+      }
+    };
+
+    const createProgressListItem = (action) => {
+      const liElem = document.createElement('li');
+      liElem.className = 'step';
+
+      const loader = document.createElement('div');
+      let loaderClass;
+      if (action.completed) {
+        loaderClass = 'done';
+      } else if (action.started) {
+        loaderClass = 'loading';
+      }
+
+      loader.className = `loader ${loaderClass}`;
+      liElem.appendChild(loader);
+
+      const span = document.createElement('span');
+      span.textContent = action.translation;
+      liElem.appendChild(span);
+
+      return liElem;
+    };
+
+    const populateList = (progress) => {
+      const actionsElem = getListElem();
+
+      Object.values(progress).forEach((action) => {
+        if (!action.display) {
+          return;
+        }
+
+        const listElement = createProgressListItem(action);
+        actionsElem.appendChild(listElement);
+      });
+    }
+
+    const getProgress = async () => {
+      const result = await fetch('/progress');
+      if (result.redirected) {
+        window.location.reload();
+        return;
+      }
+
+      if (!result.ok) {
+        console.error('Error when fetching progress', await result.text());
+        return;
+      }
+
+      const progress = await result.json();
+      clearList();
+      populateList(progress);
+    };
+
+    getProgress();
+    setInterval(getProgress, 1000);
+  </script>
+  <style>
+    html {
+      margin: 0;
+      padding: 0;
+
+    }
+    body {
+      background-color: #323232;
+      color: #FFFFFF;
+      font-size: 1em;
+      height: 100%;
+      padding: 0;
+      margin: 0;
+      overflow-y: hidden;
+
+    }
+    body, input {
+      font-family: "Noto Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+    }
```

Comment:
Copying from webapp, this should be something like...

```suggestion
    @font-face {
      font-family: 'Noto';
      src: url(/fonts/NotoSans-Regular.ttf) format("truetype");
      font-weight: normal;
    }
    body, input {
      font-family: Noto, sans-serif;
    }
```

Reply:
But.... I just realised that route ("/fonts/NotoSans-Regular.ttf") won't be available yet. Is it difficult to include it in the router?